spec/models/image_search_spec.rb

```diff
@@ -17,6 +17,46 @@
     end
   end
 
+  describe '#diagnostics' do
+    subject(:image_search) { ImageSearch.new(affiliate: affiliate, query: 'corgis', cr: use_commercial_results) }
+    let(:use_commercial_results) { nil }
+    let(:affiliate) { affiliates(:basic_affiliate) }
+    let(:search_engine) { nil }
+    before { allow(affiliate).to receive(:search_engine).and_return(search_engine) }
+    before { allow_any_instance_of(underlying_search_class).to receive(:diagnostics).and_return(:underlying_diagnostics) }
```

Comment:
Nit: this line is pretty long and could be broken up for readability

Reply:
This is also fixed, and I also broke up the `subject` business above per our chat 👍